### ISSUE TITLE
You should really really not be pushing those _vti_* folders

### ISSUE THREAD

**Issue by hal7df** [Chad414/HOT_TeamWebSite#15]
The `_vti_*` folders are entirely for the proper function of WebExpressions on the local version of the website. Since:

  1. not everyone on this repo is using WebExpressions
  1. it is only local configuration data
  1. it is constantly in flux

you can really speed up your pushes and make commits much more succinct if you tell git to ignore these folders.

In a little bit, I'll make a pull request from the fork on the hot67 repo that has all of the `_vti_*` folders removed from it, plus a .gitignore that will tell git to stop paying attention to those files (#16).

**HOWEVER**, since these folders are _already versioned_, git will delete them from everyone's local copy. For those who use WebExpressions, this is bad. Although they don't belong in the repository, they are crucial for its proper function. So, once this pull request is merged into the repository, **everyone** using WebExpressions should do the following.

 1. Make sure WebExpressions is closed. That would be bad to have it open during this process.
 1. Locate the "Git Shell" program. It should have installed to your desktop.
 1. You'll get a window with a black screen and some text. DON'T PANIC! The terminal isn't that hard to use.
 1. At the beginning of the last line you see (the prompt), there should be a file path -- it is most likely `C:`. The terminal is a bit like a file manager, it navigates through your file structure. Each folder is separated by a backslash `\`. Remember this path.
 1. Find where your repository is located relative to that path.
 1. Now, you need to change the directory that the shell is in so that you're in the repository. Type `cd` followed by a space and the relative path for the repo. For example, if your repository is in `C:\Users\paul\Documents\HOT_TeamWebSite`, then you would type `cd Users\paul\Documents\HOT_TeamWebSite`
   * If you don't want to type it all out, you can have the terminal auto-complete one directory at a time by striking `Tab` after you have typed the first few letters (it may not guess correctly, so be careful!).
 1. If you are successful, then the beginning of the prompt should now read the path of the top folder of your repository. Copy and paste this string into the shell: 

`find -type d -name "_vti*" -print0 > $FILES; git pull origin master; git checkout HEAD^ -- $FILES; git checkout master`

 In order to paste, you will have to right-click.
   * What this command does is it looks for all of the `_vti` folders and saves them into a variable. Then it gets the latest code (by pulling the code down), and then it pulls the `_vti` folders out of the previous commit.
 1. From here on out, you will have the `_vti` folders in your directory, but git will know not to pay any attention to them.

One thing: Once you merge the request, EVERYONE must do this process before you can continue with development (it will get uglier down the road if you don't). Leave your name as a response to this to indicate that you have run this operation.

I should note, that while not necessary, I would strongly encourage that you do this.

If you have any questions, just reply to this.

**Comment by hal7df** [comment]
Hmm... so apparently the Git Shell isn't complete bash...

I'm going to try and find a way to automate this process further, but in the meantime, there's something else you can do.

Once you get to step 7 in the above list, instead of copying the long text, just pull down the new code, then type `git checkout HEAD^ -- _vti_pvt/` (if you already tried the above process, just type `git reset --hard d2fa3c6` to undo everything before running the `checkout` commnad). That will get the absolutely critical folder that you need. To get the other ones back, find an option called `Recalculate hyperlinks` in WebExpressions.
